### DISCUSSION TITLE
Installing Filament into Laravel 11 --dev (pre release master)

### DISCUSSION THREAD

**Discussion by flightphysical** [filamentphp/filament#11043] [Help]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

Greetings, Newbie trying to get a bit of a head start. Attempting to install latest filament v3.2 or 3.2-dev panel builder into --dev version of Laravel 11 (master) which promises backwards compatibility to start checking for issues pre Laravel 11 official release. 

I get an installation fail message while attempting to run this command:

`
composer require filament/filament:"^3.2" -W

`
Is there a way to adjust composer.json or composer require command so that I don't get this message:

```

 Problem 1
    - filament/filament[v3.2.0, ..., 3.x-dev] require illuminate/console ^10.0 -> satisfiable by illuminate/console[v10.0.0, ..., 10.x-dev].
    - illuminate/console[v10.0.0, ..., 10.x-dev] require nunomaduro/termwind ^1.13 -> found nunomaduro/termwind[v1.13.0, ..., 1.x-dev] but these were not loaded, likely because it conflicts with another require.
    - Root composer.json requires filament/filament ^3.2 -> satisfiable by filament/filament[v3.2.0, ..., 3.x-dev].


```
 -thanks for any advice

**Comment by tremolojones** [comment]
il est compatible avec laravel 10 et non 11
